addon/services/cookies.js

```diff
@@ -30,8 +30,11 @@ export default Ember.Service.extend({
 
     return A(all).reduce((acc, cookie) => {
       if (!isEmpty(cookie)) {
-        let [key, value] = cookie.split('=');
-        acc[key.trim()] = value.trim();
+        let [key, value] = cookie.split(/=(.+)?/);
+        
+        if (key && value) {
```

Comment:
For whatever reason, an error was triggered when value == undefined. So basically, there was a cookie with an empty string so the value become undefined, thus undefined.trim() triggered an error.